lms/static/sass/elements/_controls.scss

```diff
@@ -285,6 +285,46 @@
   }
 }
 
+// imitating the pattern library
+// starts with overrides
+%btn-pl-default-base {
+  @include box-sizing(border-box);
+  @extend %t-copy-base;
+  letter-spacing: 0;
```

Comment:
NIT: any reason this property is here? is it needed for a reset of the extend above? If so, mind syncing it up with our general property order (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order)?